### ISSUE TITLE
cryptodredge ISSUES

### ISSUE THREAD

**Issue by saroshpatel09** [RainbowMiner/RainbowMiner#950]
Hi

CD starts benching and doesnt leave then localhost:4000 shows one rig as down but the benching is going on on that rig that is shown as down by localhost:4000

its been 14 minutes and it still says offline while the rig is functioning very very well
![Annotation 2020-04-12 003123](https://user-images.githubusercontent.com/32984054/79052569-17b4a300-7c55-11ea-9d26-330e79a189d2.png)
![Annotation 2020-04-12 003209](https://user-images.githubusercontent.com/32984054/79052570-18e5d000-7c55-11ea-8a64-0a7d1eee691a.png)

**Comment by RainbowMiner** [comment]
Which algo is CD mining? And, is there a CPU miner running?

**Comment by saroshpatel09** [comment]
yes there is a CPU Miner also running & CD was hashing Cyrptonight GPU at the time
restarted system re checked all but still it is the same
